src/test/rustdoc/duplicate-cfg.rs

```diff
@@ -33,6 +45,10 @@ pub mod qux {
     pub struct Qux;
 }
 
+// @has 'foo/quux/index.html'
+// @matches '-' '//*[@class="module-item"]//*[@class="stab portability"]' '^sync and send and foo and bar$'
+// @has '-' '//*[@class="module-item"]//*[@class="stab portability"]/@title' 'This is supported on crate feature `sync` and crate feature `send` and `foo` and `bar` only'
```

Comment:
nit: this would be nicer as `sync, send, foo, and bar`. But I think the existing behavior had this as well so not a blocker.